code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -45,6 +45,7 @@
 	// Small bodymass, more effect from lower volume.
 	M.adjustNutrition(nutriment_factor * (issmall(M) ? effect_multiplier * 2 : effect_multiplier)) // For hunger and fatness
 	M.add_chemical_effect(CE_BLOODRESTORE, 0.1 * (issmall(M) ? effect_multiplier * 2 : effect_multiplier))
+	M.heal_organ_damage(0.25 * effect_multiplier, 0, 5 * effect_multiplier)
```

Comment:
This is incredibly effective, look closer at the damage proc here: https://github.com/discordia-space/CEV-Eris/blob/master/code/modules/mob/living/damage_procs.dm#L103

I'd suggest `M.heal_organ_damage(0.035 * effect_multiplier, 0.035 * effect_multiplier)` as something more reasonable.

Reply:
Even a value as 0.035 could be somewhat high, perhaps to `0.025`.